modules/apps/foundation/portal-configuration/portal-configuration-test-util-test/src/testIntegration/java/com/liferay/portal/configuration/test/util/test/ConfigurationTemporarySwapperTest.java

```diff
@@ -167,8 +167,6 @@ public void testWillPreservePreviouslySavedProperties() throws Exception {
 
 		testConfiguration.update(temporaryValues);
 
-		_persistenceManager.load(_pid);
-
```

Comment:
Since updating a configuration is asynchronous, this test would intermittently fail.  This load statement was intended to "ensure" the configuration had saved before it was gotten by the test util.  If this is the wrong way to do it, I'm open to suggestions.